apps/tx-signer/test/setup-unit-tests.ts

```
@@ -1,3 +1,4 @@
+import { afterAll } from "vitest";
 import { container } from "tsyringe";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)